orchagent/copporch.cpp

```diff
@@ -10,6 +10,8 @@ using namespace swss;
 
```

Comment:
using namespace std;
and remove all the std:: in .cpp file to align with all other files.